content/docs/iac/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Pulumi IaC
```

Comment:
should the URL spell out "infrastructure-as-code"?

Reply:
No. The product "Pulumi IaC" (that's what'll be in the nav pretty much everywhere), so we're going with `iac` in the URL as well. We can add the full phrase to descriptions and headings where it makes sense, though.